app/src/test/kotlin/com/wire/android/ui/settings/devices/DeviceDetailsViewModelTest.kt

```diff
@@ -169,8 +167,8 @@ class DeviceDetailsViewModelTest {
                 arrangement.deleteClientUseCase.invoke(any()) wasNot Called
             }
             verify { arrangement.onSuccess wasNot Called }
-            assertTrue(viewModel.state.removeDeviceDialogState is RemoveDeviceDialogState.Visible)
-            assertTrue(viewModel.state.error is RemoveDeviceError.None)
+            assertEquals(true, viewModel.state.removeDeviceDialogState is RemoveDeviceDialogState.Visible)
```

Comment:
Better assertion errors if the tests break:
```suggestion
            assertInstanceOf(RemoveDeviceDialogState.Visible::class.java, viewModel.state.removeDeviceDialogState)
```

Maybe we could add kotlin-test to the project to get the Kotlin assertions like `assertIs<T>(value)`

Reply:
Cool idea! For now I added this function in `TestUtil` so that it's accessible in `app.unitTest`